### ISSUE TITLE
[CLOSED] Sigmoid warning

### ISSUE THREAD

**Issue by jeswan** [nyu-mll/jiant-v1-legacy#322]
<a href="https://github.com/sleepinyourhat"><img src="https://avatars1.githubusercontent.com/u/1284441?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [sleepinyourhat](https://github.com/sleepinyourhat)**
_Thursday Aug 02, 2018 at 13:56 GMT_
_Originally opened as https://github.com/nyu-mll/jiant/issues/322_

----

/home/sbowman/miniconda3/envs/jiant/lib/python3.6/site-packages/torch/nn/functional.py:1006: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
